index.html

```diff
@@ -2819,18 +2921,20 @@ <h5><code>subscribeallevents</code></h5>
               <code>"sse"</code>
             </li>
           </ul>
-          <p class="rfc2119-assertion" 
+	  </span>
+	  <p><span class="rfc2119-assertion" 
             id="http-sse-profile-protocol-binding-subscribeallevents-3">
             The resolved value of the <code>href</code> member MUST then be used
             as the URL of the events resource.
+	  </span>
           </p>
-          <p class="rfc2119-assertion" 
+	  <p><span class="rfc2119-assertion" 
             id="http-sse-profile-protocol-binding-subscribeallevents-2">
```

Comment:
Duplicate `id`. Not the fault of this PR but easiest to fix here.

Reply:
fixed, I think